src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -230,20 +245,15 @@ protected Task Stop()
                     }
                 });
         }
-
         protected async Task HandleShowOnlineHelpRequest(
             string helpParams,
             RequestContext<object> requestContext)
         {
-            if (helpParams == null) { helpParams = "get-help"; }
-
-            var psCommand = new PSCommand();
-            psCommand.AddCommand("Get-Help");
-            psCommand.AddArgument(helpParams);
-            psCommand.AddParameter("Online");
-
-            await editorSession.PowerShellContext.ExecuteCommand<object>(psCommand);
+            if (helpParams == null) { helpParams = "Get-Help"; }
 
+            PSCommand checkHelpPSCommand = new PSCommand();
+            checkHelpPSCommand.AddScript(checkHelpScript,true).AddArgument(helpParams);
```

Comment:
Use explicit argument name for `bool` parameters.  (VSCode has a code action for this if you put your cursor on `true` and press <kbd>CTRL + .</kbd> you should see it)

e.g.
```csharp
checkHelpPSCommand.AddScript(checkHelpScript, useLocalScope: true).AddArgument(helpParams);
```